src/app/app.component.scss

```diff
@@ -0,0 +1,10 @@
+:host {
+  display: flex;
+  flex-direction: column;
+
+  height: 100%;
+}
+
+router-outlet + ::ng-deep * {
```

Comment:
Is there any alternative to `ng-deep`? It is supposed to be deprecated soon?

Reply:
Nope so far, it's deprecated but not removed because Angular is waiting for a W3C replacement to be added to the spec.